configs/det/dbnet/db++_r50_icdar15.yaml

```diff
@@ -84,9 +84,12 @@ train:
       - RandomColorAdjust:
           brightness: 0.1255  # 32.0 / 255
           saturation: 0.5
-      - IaaAugment:
-          Fliplr: { p: 0.5 }
-          Affine: { rotate: [ -10, 10 ], p: 1.0 }
+      - RandomHorizontalFlip:
+          p: 0.5
+      - RandomRotate:
+          degrees: [ -10, 10 ]
+          expand_canvas: True
```

Comment:
Augmentation will be changed after expanding canvas, which may lead to failure in reproducing the training results on ICDAR 15. 

 expand_canvas should be False to ensure the same behavior for previous models.

Reply:
Fixed